### PR TITLE
fix: Update readme to state that in some situations scripts directory might be missing

### PR DESCRIPTION
**Pull request by ashokponkumar**
Signed-off-by: Ashok Pon Kumar <ashokponkumar@gmail.com>

### CODE REVIEW

File: assets/inbuilt/transformers/readmegenerator/templates/Readme.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ Congratulations! Move2Kube has generated the necessary build artfiacts for movin
 
 Next Steps
 ----------
-To try locally use the scripts in the "./scripts" directory, to build, push and deploy. 
+If new dockerfiles were generated by Move2Kube, scripts to build them locally are put in "./scripts" directory. 
 
-For production usage use the CI/CD pipelines for deployment.
\ No newline at end of file
+For production usage use the CI/CD pipelines and artifacts in deploy directory for deployment.
```
Comment (Akash-Nayak) [inline assets/inbuilt/transformers/readmegenerator/templates/Readme.md:9]: Can we change "deploy" directory to "./deploy" directory?

Comment (ashokponkumar) [reply assets/inbuilt/transformers/readmegenerator/templates/Readme.md]: done

Comment (Akash-Nayak) [reply assets/inbuilt/transformers/readmegenerator/templates/Readme.md]: Thanks! LGTM

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM. PS: can we keep the scripts and source in the Docker Compose case?

**Pull request merged**